File: 공예진/programmers/P181874.java
//문제: https://school.programmers.co.kr/learn/courses/30/lessons/181874
//시간: 17.90ms
//메모리: 82.5MB

class Solution {
    public String solution(String myString) {
        String answer = "";
        for (int i = 0; i < myString.length(); i++) {
            if (myString.charAt(i) == 'a') {
                answer += "A";
            } else if (myString.charAt(i) != 'A' && Character.isUpperCase(myString.charAt(i))) {
                answer += Character.toLowerCase(myString.charAt(i));
            } else {
                answer += myString.charAt(i);
            }
        }
        return answer;
    }
}

File: 강재연/programmers/P181874.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181874
// 시간: 0.11 ms
// 메모리: 33.4 MB

function solution(myString) {
    let answer = '';
    // 우선 모든 문자를 소문자로 변환 후 스프레드 연산자로 배열로 만들어서 반복문 돌림
    [...myString.toLowerCase()].forEach((str)=>{
        // 배열의 문자가 'a'일 경우에만 대문자로 변환해서 문자열에 추가
        if(str=='a') answer += str.toUpperCase();
        // 아닐경우 그냥 추가
        else answer += str;
    });
    return answer;
}
